frontend/src/pages/ApprovedActivityReport/index.js

```diff
@@ -0,0 +1,338 @@
+      setDisplayId(report.displayId);
+      setCreator(report.author.fullName);
+      setCollaborators(report.collaborators);
+      setApprovingManagers(report.approvingManager.fullName);
+      setAttendees(report.participants.join(', '));
+      setParticipantCount(report.numberOfParticipants);
+      setReasons(report.reason.join(', '));
+      setProgramType(report.programTypes.join(', '));
+      setStartDate(report.startDate);
+      setEndDate(report.endDate);
+      setDuration(report.duration);
+      setMethod(formatMethod(report.ttaType, report.virtualDeliveryType));
+      setRequester(formatRequester(report.requester));
+
+      // second table
+      setTopics(report.topics.join(', '));
+      setECLKCResources(createResourceMarkup(report.ECLKCResourcesUsed));
+      setNonECLKCResourcesUsed(createResourceMarkup(report.nonECLKCResourcesUsed));
+      setAttachments(report.attachments);
+
+      // third table
+      setContext(report.context);
+      const [goalHeadings, goals] = calculateGoalsAndObjectives(report);
+      setGoalsAndObjectiveHeadings(goalHeadings);
+      setGoalsAndObjectives(goals);
+
+      // fourth table
+      setManagerNotes(report.managerNotes);
+      setCreatorNotes(report.additionalNotes);
+    }).catch((err) => {
+      // eslint-disable-next-line no-console
+      console.log(err);
+      setSomethingWentWrong(true);
+    });
+  }, [match.params.activityReportId, user]);
+
+  async function handleCopyUrl() {
+    try {
+      await navigator.clipboard.writeText(window.location.href);
+      setSuccessfullyCopiedClipboard(true);
+    } catch (err) {
+      // eslint-disable-next-line no-console
+      console.error(err);
+    }
+  }
+
+  if (notAuthorized) {
+    return (
+      <div className="usa-alert usa-alert--error" role="alert">
+        <div className="usa-alert__body">
+          <h4 className="usa-alert__heading">Unauthorized</h4>
+          <p className="usa-alert__text">
+            Sorry, you are not allowed to view this report
+          </p>
+        </div>
+      </div>
+    );
+  }
+
+  if (somethingWentWrong) {
+    return (
+      <div className="usa-alert usa-alert--warning">
+        <div className="usa-alert__body">
+          <p className="usa-alert__text">
+            Sorry, something went wrong.
+          </p>
+        </div>
+      </div>
+    );
+  }
+
+  return (
+    <>
+      {successfullyCopiedClipboard ? (
+        <div className="usa-alert usa-alert--success margin-bottom-2">
+          <div className="usa-alert__body">
+            <p className="usa-alert__text">Successfully copied URL</p>
+          </div>
+        </div>
+      ) : null }
+      <Grid row>
+        <button type="button" className="usa-button no-print" onClick={handleCopyUrl} style={{ display: navigator.clipboard ? 'inline' : 'none' }}>Copy URL Link</button>
+        <button type="button" className="usa-button no-print" onClick={() => window.print()}>Print to PDF</button>
```

Comment:
I would think they would want the Creator and Manager notes hidden on the print here as well.